gh-pages/html-preview/index.html

```diff
@@ -62,6 +62,9 @@
             }
 
             function onError(err) {
+              const msg =
```

Comment:
Can we add a check against the userAgent to check for `brave`? I think something like `navigator.userAgent.IndeOf('Brave') !== -1` might do the trick

Reply:
so it doesnt look weird for non Brave users. For non Brave users it might be more usedul to do alert(err) so if an issue actually happens, people can add the error into new issues. or a console.log(err) for both cases? whatever you think its best.

Reply:
Good call, just pushed an update that does this:
```javascript
const msg = navigator.brave
  ? 'There was an error generating the preview. If you are using Brave browser, please turn off shields and try again, or use a different browser.'
  : 'There was an error generating the preview. Please check the web console for errors.'
alert(msg)
```

The error already prints to web console so no need to add here.